tools/ci-ts/test-helpers/common.ts

```diff
@@ -0,0 +1,255 @@
+import { Compiler } from '@0x/sol-compiler'
```

Comment:
Could we merge this all into `integration-scripts`?

Reply:
@HenryNguyen5  I think we ought to be able to fully combine `integration_scripts`, `tools/ci`, and `tools/ci-ts` into a single ci directory once the other tests are converted to TS also. My thought was to leave them separate for now but then once we convert the ethLog and runLog tests we could squash them into a single directory. Thoughts?